ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala

```diff
@@ -1,20 +1,27 @@
 // Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.metrics
+package com.daml.http.metrics
 
 import com.codahale.metrics.MetricRegistry
+import com.daml.metrics.CacheMetrics
 import com.daml.metrics.api.MetricHandle.{Counter, Meter, Timer}
 import com.daml.metrics.api.MetricName
 import com.daml.metrics.api.dropwizard.DropwizardFactory
 import com.daml.metrics.api.opentelemetry.OpenTelemetryFactory
+import io.opentelemetry.api.GlobalOpenTelemetry
 import io.opentelemetry.api.metrics.{Meter => OtelMeter}
 
+object HttpJsonApiMetrics {
+  lazy val ForTesting =
+    new HttpJsonApiMetrics(new MetricRegistry, GlobalOpenTelemetry.getMeter("test"))
+}
+
 class HttpJsonApiMetrics(
-    val prefix: MetricName,
     val registry: MetricRegistry,
     val otelMeter: OtelMeter,
-) {
+) extends DropwizardFactory {
```

Comment:
What are the reasons for `HttpJsonApiMetrics` to extend `DropwizardFactory` ?

Reply:
No reason, I overlooked that the subsequent timers are created by a `DropwizardFactory` defined into the class.

Reply:
Removing it.